docs/install/vm-install/suse.md

```diff
@@ -27,6 +27,10 @@ sudo firewall-cmd --permanent --zone=public --add-service=https
 ```shell
 sudo firewall-cmd --reload
 ```
+- for Suse Linux we need to enable PackageHub as per OSversion and architecture
```

Comment:
Instead of `SUSE Linux`, using `SUSE Linux Enterprise(SLES)` would be better. This will make it clear that this instruction is not for OpenSUSE.

Reply:
Also, change `OSversion` to `OS version`

Reply:
made changes according to suggestion